src/examples/button2/button2.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"machine"
+	"time"
+)
+
+// This example assumes that you are using the pca10040 board
+
+func setLED(led *machine.GPIO, state bool) {
```

Comment:
I think you forgot to remove this function?

Reply:
OK, updated.